rerun_py/rerun_sdk/rerun/sinks.py

```diff
@@ -25,6 +26,55 @@ def connect(
     flush_timeout_sec: float | None = 2.0,
     default_blueprint: BlueprintLike | None = None,
     recording: RecordingStream | None = None,
+) -> None:
```

Comment:
Please add `@deprecated` decorator as in:
https://github.com/rerun-io/rerun/pull/3564